py/test_util.py

```diff
@@ -1,32 +1,107 @@
 import logging
+import subprocess
+import time
 from xml.etree import ElementTree
 
 import six
 
 from py import util
 
+# TODO(jlewi): How can we indicate that a test wasn't run?
 class TestCase(object):
```

Comment:
library suggestion: https://github.com/python-attrs/attrs or namedtuple

Reply:
I meant in the junit.xml file that is created and uploaded to gubernator.
I ended up marking the test as failed with a failure message of "Test not run."
PTAL>